external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/package.scala

```diff
@@ -17,7 +17,52 @@
 
 package org.apache.spark.streaming
 
+import org.apache.spark.internal.config.ConfigBuilder
+
 /**
  * Spark Integration for Kafka 0.10
  */
-package object kafka010 //scalastyle:ignore
+package object kafka010 { //scalastyle:ignore
+
+  private[spark] val CONSUMER_CACHE_ENABLED =
+    ConfigBuilder("spark.streaming.kafka.consumer.cache.enabled")
+      .booleanConf
+      .createWithDefault(true)
+
+  private[spark] val CONSUMER_POLL_MS =
+    ConfigBuilder("spark.streaming.kafka.consumer.poll.ms")
+    .longConf
```

Comment:
Why not `.timeConf(TimeUnit.MILLISECONDS)`?

Reply:
Good question; unfortunately the key has ".ms" so might look funny to specify `spark.streaming.kafka.consumer.poll.ms=10s`. I don't mind leaving it as-is for now as I think we took the same stance with other similar situations (?)

Reply:
> Why not `.timeConf(TimeUnit.MILLISECONDS)`?

I think both method is good, so I think It is not necessary to use this.

Reply:
> Good question; unfortunately the key has ".ms" so might look funny to specify `spark.streaming.kafka.consumer.poll.ms=10s`. I don't mind leaving it as-is for now as I think we took the same stance with other similar situations (?)

I think both method is good, so I think It is not necessary to use `timeConf`.